app/models/document.rb

```diff
@@ -27,6 +27,7 @@
 
 class Document < ActiveRecord::Base
   belongs_to :user, :autosave => true
+  belongs_to :anthology
```

Comment:
It has to be possible for documents to belong to any number of anthologies -- this should be a `has_many`